service-print/src/main/java/org/oskari/print/PDF.java

```diff
@@ -321,6 +336,79 @@ private static void drawTimeseriesTexts(PDPageContentStream stream,
                 x, OFFSET_TIME_IN_TIMESERIES_BOTTOM);
     }
 
+    private static void drawCornerCoords(PDPageContentStream stream, PrintRequest request, float x, float y, float mapHeight, float mapWidth, PDRectangle pageSize)
```

Comment:
Page size isn't used so we can remove it from params:

```suggestion
    private static void drawCornerCoords(PDPageContentStream stream, PrintRequest request, float x, float y, float mapHeight, float mapWidth)
```